bookmarks.js

```diff
@@ -1,3 +1,21 @@
+import { getSections } from './styles/lib/db.js'
+import { createCard } from './styles/lib/card.js'
+
+const content = document.querySelector('.container__content')
+
+const data = getSections()
+
+const filteredCards = createCard.filter((bookmark) => {
```

Comment:
This should not work. You are trying to run a filter method on `createCard` function. (You imported it in `line2`) You want the filter method (it's an `array` method) applied on an array. ;-)